src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/stash/StashApiClient.java

```diff
@@ -197,21 +203,23 @@ private HttpClient getHttpClient() {
     httpParams.setParameter(
         CoreConnectionPNames.SO_TIMEOUT, StashApiClient.HTTP_SOCKET_TIMEOUT_SECONDS * 1000);
 
-    //        if (Jenkins.getInstance() != null) {
-    //            ProxyConfiguration proxy = Jenkins.getInstance().proxy;
-    //            if (proxy != null) {
-    //                logger.info("Jenkins proxy: " + proxy.name + ":" + proxy.port);
-    //                client.getHostConfiguration().setProxy(proxy.name, proxy.port);
-    //                String username = proxy.getUserName();
-    //                String password = proxy.getPassword();
-    //                // Consider it to be passed if username specified. Sufficient?
-    //                if (username != null && !"".equals(username.trim())) {
-    //                    logger.info("Using proxy authentication (user=" + username + ")");
-    //                    client.getState().setProxyCredentials(AuthScope.ANY,
-    //                        new UsernamePasswordCredentials(username, password));
-    //                }
-    //            }
-    //        }
+    if (Jenkins.getInstanceOrNull() != null && proxy != null) {
+      logger.info("Using proxy: " + proxy.name + ":" + proxy.port);
```

Comment:
`logger.info` is enabled but default. It would log something every time Bitbucket is checked for new/updates PRs. That would be a lot. Please use `logger.fine` or just remove that logging. If it's not useful for the administrator, it should not be logged. The same applies to the `logger.info` below.

Reply:
Done